Scheduler Grouping: rewrite (#877)
DevExpress/devextreme-demos: JSDemos/Demos/Scheduler/GroupOrientation/description.md

> If [resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are assigned to appointments, you can group them by these resources. To do this, specify the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. Note that the order of resource fields depends on the order of elements in the **resources** array.

Reviewer: Let's look at the description from the standpoint of a reader who doesn't know about resource. We should first introduce him to the concept, give a link to the Resources demo, and only then go on about grouping.

> If resources are assigned to appointments, you can group them by these resources. To do so, use the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property to specify an array. Each element of this array is **fieldExpr** of a resource kind. Note that the resource headers' order depends on the resources' order in the [resources](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/resources '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/') array.

Reviewer 2: >If resources are assigned to appointments, you can group them by these resources. 

Please add links to appointments and resources.

>To do so, use the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') property to specify an array. 

This sentence breaks suddenly. How about changing it a bit as follows: To do this, specify the [groups[]]() array. 

>Each element of this array is **fieldExpr** of a resource kind. 

Let's add a link.
 
>Note that the resource headers' order depends on the resources' order in the [resources]() array.

Once again, I suggest you not to use apostrophes with plurals. How about this: Note that the order of resource headers depends on the order of elements in the **resources[]** array.

> The Scheduler arranges group headers vertically (in a column) or horizontally (in a row). This depends on the view type. You can use the **views**.[groupOrientation](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/groupOrientation.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation') property to set custom orientation for specific views.

Reviewer 2: ```suggestion
The Scheduler arranges group headers vertically (in a column) or horizontally (in a row). This depends on the view [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type). You can use the **views**.[groupOrientation](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/groupOrientation.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation') property to set custom orientation for specific views.
```

> The Scheduler arranges group headers vertically (in a column) or horizontally (in a row). This depends on the view [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type). You can use the **views**.[groupOrientation](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/groupOrientation.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation') property to set custom orientation for specific views.

Reviewer: Use only URLs in links within demo descriptions.

> Appointments can be grouped by resources. This demo declares a single resource&mdash;Priority. Refer to the following demo for more information about resources and their configuration: [Resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).

Reviewer 3: ```suggestion
Users can group appointments by resources. This demo declares a single resource&mdash;Priority. Refer to the following demo for more information about resources and their configuration: [Resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
```

> [Resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are categories used to classify appointments. To understand what resources are, consider the following example: in an educational center lectures are held in several rooms; in Scheduler terms, room is a resource kind, individual rooms are resource instances, and lectures are appointments that use these resource instances.

Reviewer: We do need to make a connection between grouping and resources. However, this demo is about grouping, and we shouldn't dive deep into explaining the resources concept. The following description would suffice:

```suggestion
Appointments can be grouped by resources. This demo declares a single resource&mdash;Priority. Refer to the following demo for more information about resources and their configuration: [Resources](/Demos/WidgetsGallery/Demo/Scheduler/Resources/).
```

> To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of group headers in the UI mirrors the order of resource instances. If the **groups[]** array contains more than one element, groups are nested. 

Reviewer 3: ```suggestion
To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is the [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a corresponding resource kind. The order of group headers in the UI mirrors the order of resource instances in the array. If the **groups[]** array contains more than one element, groups are nested. 
```

> To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is the [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a corresponding resource kind. The order of group headers in the UI mirrors the order of items in the resource instances array. If the **groups[]** array contains more than one element, groups are nested. 

Reviewer 4: ```suggestion
To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is the [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a corresponding resource kind. The order of group headers in the UI is the same as items in the resource instances array. If the **groups[]** array contains more than one element, groups are nested. 
```

> The [Resources demo](/Demos/WidgetsGallery/Demo/Scheduler/Resources/) describes how to assign resources. When assigned, they can also be used to group appointments. To implement this, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of resource headers in the UI is the same as the order of the resource instances. Note that if the **resources** array has more than one element, grouping is nested. 

Reviewer: ```suggestion
To group appointments by resources, specify the [groups[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups) array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. The order of group headers in the UI mirrors the order of resource instances. If the **groups[]** array contains more than one element, groups are nested. 
```

Reviewer: > The order of resource headers in the UI is the same as the order of the resource instances.

By "resource headers" you mean "group headers"? "the order of the resource instances" in which array?

> Note that if the **resources** array has more than one element, grouping is nested. 

Do you mean "if the **groups** array has more than one element"?

> The Scheduler can arrange group headers vertically (in a column) or horizontally (in a row). You can use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set custom orientation for specific views. This demo shows two Work Week views, each of which has a different group orientation.

Reviewer 3: ```suggestion
The Scheduler can arrange group headers vertically (in a column) or horizontally (in a row). Use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set a custom orientation for specific views. This demo shows two Work Week views with different group orientations.
```

> The Scheduler arranges group headers vertically (in a column) or horizontally (in a row). You can use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set custom orientation for specific views. This property description includes default view orientation types.

Reviewer: > The Scheduler arranges

Without "can", it reads as if the Scheduler chooses group orientation randomly.

> This property description includes default view orientation types.

It goes without saying. We better refer to the demo instead.

```suggestion
The Scheduler can arrange group headers vertically (in a column) or horizontally (in a row). You can use the **views**.[groupOrientation](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#groupOrientation) property to set custom orientation for specific views. This demo shows two Work Week views, each of which has a different group orientation.
```
